app/controllers/ReportController.java

```diff
@@ -101,17 +165,21 @@ public Result query(
      * @return Matching report.
      */
     public Result get(final String id) {
-        return notFound();
-    }
-
-    /**
-     * Delete a specific report.
-     *
-     * @param id The identifier of the report.
-     * @return No content if successful, otherwise an HTTP error code.
-     */
-    public Result delete(final String id) {
```

Comment:
Why was delete removed?

Reply:
There isn't currently a delete method on the repository. I can add that in and plumb it through here if you'd like that in this PR, or I can handle that afterwards.

Reply:
After is fine.